sdk/eventhub/event-hubs/src/managementClient.ts

```diff
@@ -414,13 +415,11 @@ export class ManagementClient extends LinkEntity {
           }
         });
 
-      const maxRetries = options.retryOptions && options.retryOptions.maxRetries;
+      const maxRetries = retryOptions.maxRetries;
       const delayInSeconds =
-        options.retryOptions &&
-        options.retryOptions.retryInterval &&
-        options.retryOptions.retryInterval >= 0
-          ? options.retryOptions.retryInterval / 1000
-          : Constants.defaultDelayBetweenOperationRetriesInSeconds;
+        typeof retryOptions.retryInterval === "number"
+          ? retryOptions.retryInterval / 1000
+          : undefined;
```

Comment:
The above 2 constants are not needed. They can be inlined just like we do in other places where we create the `RetryConfig`. Then as part of #4401, the inline will get simplified too